### DISCUSSION TITLE
chat teams

### DISCUSSION THREAD

**Discussion by EricYB** [Launch-X-Latam/MisionFrontEnd#66] [Q&A]
al alguien mas no le daja escribir en el chat que al menos a mi no me deja

**Comment by EduGarcia502** [comment]
xd me pasa lo mismo me siento marginado :V
<h1>que mal</h1>

**Comment by JosafatJimenezB** [comment]
Solo den click donde dice ¿haga una pregunta? y ahi escriban lo que quieran que aparezca en el chat

> recuerden no hacer anonimas sus preguntas

**Reply by ArturoEA** [reply]
Gracias!

**Reply by JosafatJimenezB** [reply]
denada master

**Comment by LaMazorca** [comment]
Gracias, ese problema igual tengo

**Comment by OscarNoe** [comment]
Para escribir en el "chat" debes dar click en el icono de la burbuja con el signo de interrogación:
![image](https://user-images.githubusercontent.com/41075783/192910924-3ea9b1a6-98e7-4bbb-a77f-694165799bb9.png)
y despues escribirlo como pregunta:
![image](https://user-images.githubusercontent.com/41075783/192910966-5cf4a488-aa89-428b-bd35-acc95cb8b6ce.png)

**Reply by OscarNoe** [reply]
El boton de pregunta se encuenta en la esquina superior derecha

**Reply by JosafatJimenezB** [reply]
confirmo

**Reply by EricYB** [reply]
lo ubico pero no me deja igual jaja

**Reply by JosafatJimenezB** [reply]
si no te deja @EricYB solo vuelve a recargar teams, eso nunca falla

**Reply by EricYB** [reply]
va okey gracias... @JosafatJimenezB

**Reply by JosafatJimenezB** [reply]
denada crack

**Comment by OATS74** [comment]
Creo que fui baneado por querer comentar en el live del miércoles 28. Y hasta ahora no he recibido un pase al team  . Ya que es por ese grupo que pasan el link para el streaming

**Comment by Amhedriel** [comment]
Queridos compañeros, todos podemos utilizar el chat, sin embargo a nosotros nos mostrará únicamente los mensajes que nosotros escribamos, el Mission Commander podrá compartir los chats más destacados, en el cual podremos verlos todos.

Ahora, posiblemente estén (o yo esté confundido y lo sepan) pero, el chat que nosotros vimos en la transmisión del miércoles es el chat principal que solo el que hace el Live puede ver, pero nos hizo el favor de mostrarlo en pantalla para que nosotros también podamos verlo, puede escoger entre esos mensajes y compartirlos con todos nosotros en el chat de destacados.

Abrazos.
